client/python/lib/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+VIRTUALENV_EXE := python3 -m virtualenv -p python3
+VIRTUALENV_DIR := .venv
+ACTIVATE="$(VIRTUALENV_DIR)/bin/activate"
+
+.PHONY: build-deps install-package test clean install
+
+venv:$(ACTIVATE)
+	@echo -n "Using virtual environment: $(VIRTUALENV_DIR)\n"
+	@. $(ACTIVATE); python3 --version
+
+$(ACTIVATE):
+	@echo "Updating virtualenv dependencies in: $(VIRTUALENV_DIR)..."
+	@test -d $(VIRTUALENV_DIR) || $(VIRTUALENV_EXE) $(VIRTUALENV_DIR)
+	@. $(ACTIVATE); pip install --upgrade pip
+	@. $(ACTIVATE); pip install -r tests/requirements.txt
+	@touch $(ACTIVATE)
+
+
+build-deps: venv
+	@echo "Building dependencies"
+	@sudo apt install -y protobuf-compiler
```

Comment:
I don't think it would be good approach to add root commands to the makefile scope

Reply:
`protobuf-compiler` needs to be installed in order to build TFS API.
Do you think we should specify it as a prerequisite instead of using it with make call?